src/utils/components/CPUMemoryModal/components/CPUInput/components/vCPUInput/components/CPUHelperText/CPUHelperText.scss

```diff
@@ -0,0 +1,8 @@
+div#cpu-helper-text {
```

Comment:
is there a reason why the div is needed? id should be unique

Reply:
At one point in testing it wasn't working without it, but I checked again and it worked fine, so I removed it. =)